service/src/main/java/com/epam/esm/exception/ValidationError.java

```diff
@@ -4,10 +4,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class ValidationException extends RuntimeException {
+public class ValidationError extends RuntimeException {
     private final Map<String, String> validations;
```

Comment:
What I mean was not to rename exception, it's ok. Field validations is not expected inside of exceptions and makes you draw attention to it. When you see ValidationException you expect something like validationErrors inside, not validations (what does it mean validation results at all)